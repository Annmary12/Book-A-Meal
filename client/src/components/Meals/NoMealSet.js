import React from 'react';
import pot from '../../../assets/img/pot.svg';
import styles from './no-meal-set.scss';

const NoMealSet = () => (
  <div className={styles.container}>
    <img src={pot} width="150" alt="meal cooking" />
    <p className={styles.text}>No meal has been added.</p>
  </div>
);

export default NoMealSet;
