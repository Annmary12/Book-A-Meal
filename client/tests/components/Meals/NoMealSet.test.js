import React from 'react';
import NoMealSet from '../../../src/components/Meals/NoMealSet';

describe('NoMealSet component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(
      <NoMealSet />
    );

    expect(wrapper).toMatchSnapshot();
  });
});