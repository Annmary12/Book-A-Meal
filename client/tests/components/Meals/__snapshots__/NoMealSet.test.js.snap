// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMealSet component should match snapshot 1`] = `
<div
  className="container"
>
  <img
    alt="meal cooking"
    src="test-file-stub"
    width="150"
  />
  <p
    className="text"
  >
    No meal has been added.
  </p>
</div>
`;
